neqo-http3/src/connection.rs

```diff
@@ -1284,7 +1288,7 @@ impl Http3Connection {
             .map_err(|e| Error::map_stream_create_errors(&e))?;
         // Set outgoing WebTransport streams to be fair (share bandwidth)
         // This really can't fail, panics if it does
-        conn.stream_fairness(stream_id, true).unwrap();
+        conn.stream_fairness(stream_id, true)?;
```

Comment:
Comment is outdated now. Also I would be fine with a `stream_fairness(stream_id, true).expect("~this really can't fail ...");`